testing/btest/Baseline/scripts.base.frameworks.cluster.cluster_init/manager-1..stdout

```diff
@@ -0,0 +1,8 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+### NOTE: This file has been sorted with diff-sort.
+Got cluster_init event
```

Comment:
I'm confused. The way I read the docs for the events, `node_connected` should be sent once a node has connected to all of its peers and `cluster_init` should be sent after all of the nodes report that they're connected to all of their peers. Shouldn't that means `cluster_init` happens at the end here?

Reply:
You're right. However, the output is sorted as the sequence of `node_connected` might be arbitrary. Probably better to skip the node names in the output so we don't need the sorting.